app/forms/responsiblepeople/PersonUKPassportFormProvider.scala

```diff
@@ -38,13 +38,15 @@ class PersonUKPassportFormProvider @Inject()() extends Mappings {
       booleanFieldName -> boolean(booleanError, booleanError),
       "ukPassportNumber" -> mandatoryIfTrue(
         booleanFieldName,
-        text(s"$booleanError.number").verifying(
-          firstError(
-            maxLength(length, lengthError),
-            minLength(length, lengthError),
-            regexp(regex, "error.invalid.uk.passport")
+        textAllowWhitespace(s"$booleanError.number")
+          .transform[String](_.replace(" ", "").trim, x => x)
```

Comment:
Line 42 is no longer needed if you're using `textAllowWhitespace` so it can be removed

Reply:
Resolved